### PR TITLE
02.Gil.local + 03.Git.Hosting v2

### PR DESCRIPTION
**Pull request by BrazovskyVladimir**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+image: ubuntu
```
Comment (pluhin) [inline .gitlab-ci.yml]: All tour files should be in your folder, can you move it

File: VladimirBrazovsky/03.Git.Hosting/03.Git.hosting.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+orig_bit_braz	ssh://git@bitbucket.org/brazovsky/brazovsky.git (fetch)
```
Comment (pluhin) [inline VladimirBrazovsky/03.Git.Hosting/03.Git.hosting.md]: Please, use git markup for this addresses

**Comment by BrazovskyVladimir** [comment]
I changed

**Comment by BrazovskyVladimir** [comment]
I replaced 02.Git.local to README.md in 02.Git.local folder. But I made mistake when created first repository and my "master" branch hid under your "master" after rebase. I renamed my branch, but all commit were lose. I can show screenshot from GitKraken.   (Пишу в спешке, прошу простить за кривой English)

**Pull request merged**